### DISCUSSION TITLE
Where to find throat.centroid?

### DISCUSSION THREAD

**Discussion by kweishaupt** [PMEAL/porespy#554] [Q&A] [answered]
Around half a year ago on the dev branch, I was able to get `throat.centroid` from an extracted network using `geo['throat.centroid']`.

Has this property been renamed or removed? Is it the same as `throat.global_peak`?

**Comment by jgostick** [comment, marked as answer]
Yes, it's been renamed global_peak, which is the location of the maxima of the global distance transform within the throat voxels (defined by the 2-layer thick region forming the boundary where the two neighboring regions touch).  This has always been how the centroid was found, so we changed the name to be more descriptive.  It would be easy to add the centroid, but it's calculated in a different way (it's the actual geometric centroid of the voxels)
